stages/org.osbuild.systemd.preset

```diff
@@ -28,14 +28,31 @@ SCHEMA = r"""
       }
     },
     "description": "Array of systemd unit names and their preset logic."
+  },
+  "user-presets": {
+    "type": "array",
+    "minItems": 1,
+    "items": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["name", "state"],
+      "properties": {
+        "name": {"type": "string"},
+        "state": {
+          "type": "string",
+          "enum": ["enable", "disable"]
+        }
+      }
+    },
+    "description": "Array of systemd unit names and their preset logic."
```

Comment:
nitpick: Please explicitly state in this description that this is for user units and also please amend the description of the `presets` property to explicitly state that it is for system units.